docs/source/Analysis/queries.md

```diff
@@ -202,26 +172,22 @@ Running this algorithm returns the following data:
 32670,Celebrimbor,2
 32670,Grimbeorn,2
 32670,Lobelia,2
+...
 ```
 
-This data tells us that at a given time, person X and Gandalf are N number of hops away.
-For example, at time 32670, Samwise was at minimum 1 hop away from Gandalf, whereas Lobelia was 2 hops away.
+This data tells us that at a given time, person X and Gandalf are N number of hops away. In this instance, at time 32670, Samwise was at minimum 1 hop away from Gandalf, whereas Odo was 2 hops away.
 
 ## Using Raphtory as a client
 
-Finally, if you have a graph deployed somewhere else and want to submit new queries to it,
-you can do this via the `deployedGraph(customConfig)` method in `Raphtory` object,
-providing the appropriate configuration to locate it.
-If the graph is deployed in the same machine using the default configuration,
-you can even omit the configuration parameter:
+Finally, if you have a graph deployed somewhere else and want to submit new queries to it you can do this via the `deployedGraph(customConfig)` method in the `Raphtory` object. The `customConfig` here is to provide the appropriate configuration to locate the graph (i.e. the akka/pulsar address). If the graph is deployed in the same machine using the default Raphtory configuration you can omit this configuration parameter:
 
 ```scala
 val graph = Raphtory.deployedGraph()
 ```
 
 From this point, you can keep working with your graph as we have done so far.
 
-Additionally, you still have access to the `RaphtoryClient` class provided in previous releases of Raphtory:
+Additionally, you still have access to the `RaphtoryClient` class provided in previous releases of Raphtory. This is, however, deprecated and will be removed in later versions:
 
 ```scala
 val client = Raphtory.createClient()
```

Comment:
Are we going to remove this finally?

Reply:
Yeah probably can do, I don't believe anyone is using it